python/spotify.py

```diff
@@ -43,48 +45,66 @@
 #     version 0.1: initial
 #
 
-import re
 import datetime
-import weechat as w
+import re
+
 import spotipy
+import weechat as w
 from spotipy.oauth2 import SpotifyClientCredentials
 
-SCRIPT_NAME    = "spotify"
-SCRIPT_AUTHOR  = "xt <xt@bash.no>"
-SCRIPT_VERSION = "0.8"
+SCRIPT_NAME = "spotify"
+SCRIPT_AUTHOR = "xt <xt@bash.no>"
+SCRIPT_VERSION = "0.9"
 SCRIPT_LICENSE = "GPL"
-SCRIPT_DESC    = "Look up spotify urls"
+SCRIPT_DESC = "Look up spotify urls"
 
 settings = {
-    "buffers"       : 'freenode.#mychan,',     # comma separated list of buffers
-    "emit_notice"   : 'off',                   # on or off, use notice or msg
-    "client_id"     : 'client_id',
-    "client_secret" : 'client_secret'
+    "buffers": 'freenode.#mychan,',  # comma separated list of buffers
+    "emit_notice": 'off',  # on or off, use notice or msg
+    "client_id": 'client_id',
+    "client_secret": 'client_secret'
 }
 
 settings_help = {
-    "buffers"       : 'A comma separated list of buffers the script should check',
-    "emit_notice"   : 'If on, this script will use /notice, if off, it will use /msg to post info',
-    "client_id"     : 'required client id token go to https://developer.spotify.com/my-applications/#!/applications to generate your own',
-    "client_secret" : 'required client secret token go to https://developer.spotify.com/my-applications/#!/applications to generate your own'
+    "buffers":
+    'A comma separated list of buffers the script should check',
+    "emit_notice":
+    'If on, this script will use /notice, if off, it will use /msg to post info',
+    "client_id":
+    'required client id token go to https://developer.spotify.com/my-applications/#!/applications to generate your own',
```

Comment:
What was your line limit restriction when reformatting the code? If it was 80 this exceeds said limit.